apps/condo/domains/ticket/utils/unit.js

```diff
@@ -13,7 +13,7 @@ function getSectionAndFloorByUnitName (property, unitName, unitType) {
                 for (const unit of floor.units) {
                     if (unit.label === unitName) {
                         res.sectionName = section.name
-                        res.sectionType = section.type
+                        res.sectionType = unitType !== 'parking' ? section.type : 'parking'
```

Comment:
It looks strange. Probably you need to talk with @sitozzz .
Probably, we need to fix all map data for that case.

Reply:
@pahaz that's because `section.unitType` field is not required at `Ticket` domain:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/25844927/188646848-994f1e4e-c950-4183-b56e-505ecf08f58f.png">

So this lines of code is used for determining section attributes by unit